### DISCUSSION TITLE
@percy/cli no longer supports valid props like storybook

### DISCUSSION THREAD

**Discussion by brainbrian** [percy/cli#785] [Support] [answered]
## The problem

As mentioned in the title, the latest version of the package `@percy/cli@1.0.0-beta.75` appears to have dropped support for config props like `upload` and `storybook`. Version `1.0.0-beta.74` supports these props.

## Environment

- `@percy/cli` version: 1.0.0-beta.75
- Version of Percy SDK you’re using: `@percy/storybook@4.1.0`

## Details

Running `percy config:validate` throws the following errors:
```
❯ percy config:validate            
[percy] Found config file: .percy.js
[percy] Invalid config:
[percy] - snapshot: unknown property
[percy] - upload: unknown property
[percy] - storybook: unknown property
```

### Sample config
```
require('dotenv').config();

module.exports = {
  version: 2,
  snapshot: {
    widths: [1280],
    minHeight: 1024,
    percyCSS: `.percy-hide { visibility: hidden; }`,
  },
  upload: {
    files: '**/*.{png,jpg,jpeg}',
    ignore: '',
  },
  storybook: {
    include: [
      'ComponentOne',
      'ComponentTwo',
    ],
  },
};
```

**Comment by Robdel12** [comment, marked as answer]
Hey @brainbrian! That's interesting given the only change between the two versions was changing an import to a require in the `bin/run` file. I'm also not able to reproduce on a fresh project setup. Could you provide a reproduction? 

My hunch is there's a dependency resolution issue. Deleting `node_modules` and installing from scratch might correct this.

**Reply by brainbrian** [reply]
I believe you are correct. After removing both `@percy/cli` and `@percy/storybook` and reinstalling both the config validates as expected. I think there were 2 different resolutions for `@percy/cli-command` in my old lock.

I also confirmed this first by standing up a simple test.

package.json
```{
  "name": "percy-test",
  "scripts": {
    "validate": "percy config:validate"
  },
  "devDependencies": {
    "@percy/cli": "1.0.0-beta.75",
    "@percy/storybook": "^4.1.0"
  }
}
```

.percy.js
```module.exports = {
  version: 2,
  snapshot: {
    widths: [1280],
    minHeight: 1024,
    percyCSS: `.percy-hide { visibility: hidden; }`,
  },
  upload: {
    files: '**/*.{png,jpg,jpeg}',
    ignore: '',
  },
  storybook: {
    include: [
      'ComponentOne',
      'ComponentTwo',
    ],
  },
};
```

```
❯ npm run validate

> validate
> percy config:validate

[percy] Found config file: .percy.js
[percy] Using config:
{
  version: 2,
  snapshot: {
    widths: [
      1280
    ],
    minHeight: 1024,
    percyCSS: '.percy-hide { visibility: hidden; }'
  },
  upload: {
    files: '**/*.{png,jpg,jpeg}',
    ignore: ''
  },
  storybook: {
    include: [
      'ComponentOne',
      'ComponentTwo',
    ]
  }
}
```

Thank you for the suggestion.

**Reply by Robdel12** [reply]
I wish it wasn't true 😭 I'm not sure there's much we can do to fix the package managers dependency resolution but we'll keep this in mind and maybe an idea will come up eventually.

**Reply by brainbrian** [reply]
If it helps, I got myself into this with `npm update`.

```
❯ node -v && npm -v
v14.17.4
6.14.14
```
